docs/introduction/installation.md

```diff
@@ -125,16 +125,8 @@ config :vachan, Vachan.Repo,
 Create and migrate the database:
 
 ```sh
-mix ecto.create
-mix ecto.migrate
-```
-
-### 5. Install Additional Tools (Optional)
-
-If you need any additional tools or dependencies like Phoenix LiveDashboard or Ash Admin, install them as follows:
-
-```sh
-mix archive.install hex phx_new
+mix ash_ecto.create
```

Comment:
mix ash_postgres.create

reference: https://hexdocs.pm/ash_postgres/migrations-and-tasks.html#ashpostgres-specific-mix-tasks